chat-commands.js

```diff
@@ -3300,6 +3300,7 @@ exports.commands = {
 		}
 		if (!target) return this.errorReply("Provide a valid format.");
 		let originalFormat = Tools.getFormat(target);
+		// Client handles sending a default format if none is selected.
```

Comment:
Something more like:

```
// Note: The default here of [Gen 7] Pokebank Anything Goes isn't normally hit; since the web client will send a default format
```